xonsh/tools.py

```diff
@@ -1171,11 +1171,22 @@ def ensure_string(x):
     return str(x)
 
 
+def is_path(x):
+    """This tests if something is a path."""
+    return isinstance(x, pathlib.Path)
+
+
 def is_env_path(x):
     """This tests if something is an environment path, ie a list of strings."""
     return isinstance(x, EnvPath)
 
 
+def str_to_path(x):
```

Comment:
Can you please add some tests to test_tools.py for the functions you added here.

Reply:
Done )